requirements_lock.txt

```diff
@@ -1,63 +1,60 @@
 alabaster==0.7.12
-astroid==2.8.0
-attrs==21.2.0
+astroid==2.9.3
+attrs==21.4.0
 Babel==2.9.1
-black==21.9b0
-certifi==2021.5.30
-charset-normalizer==2.0.6
-click==8.0.1
+black==22.1.0
+certifi==2021.10.8
+charset-normalizer==2.0.11
+click==8.0.3
 click-default-group==1.2.2
 contexttimer==0.3.3
-coverage==6.0.1
-dataclasses==0.8
```

Comment:
was datac;asses unused?

Reply:
dataclasses are built-in to python 3.7 so the backwards compatibility package can be dropped.